src/service/resources-impl.js

```diff
@@ -794,7 +794,10 @@ export class Resources {
             minTop = minTop == -1 ? box.top : Math.min(minTop, box.top);
           }
           request.resource./*OK*/changeSize(
-              request.newHeight, request.newWidth, request.callback);
+              request.newHeight, request.newWidth);
+          if (request.callback) {
+            request.callback(/* hasSizeChanged */true);
```

Comment:
I found I can't call at the end of this conditional because with `resize==false` I can't just call `callback(false)`, the callback could still be called after scroll.
If I don't want to call it in several places,  I'll have to keep a record of the reason why this request is rejected to determine reject or resolve.
@jridgewell  Any suggestions?